### ISSUE TITLE
paper idea

### ISSUE THREAD

**Issue by dongzhi0312** [postBG/DTA.pytorch#11]
SAdD is used to maximize the divergence between a model’s prediction
and ground truth label.
our goal is to enforce the cluster assumption on target data by minimizing the divergence between predictions.
there have conflict？

**Comment by numpee** [comment]
Hi, thanks for your interest in our work.

With adversarial training methods (such as VAT, SAdD, VAdD, and even the original adversarial attack paper) the goal is to use these adversarial perturbations to help with training. In other words, the samples are made to be adversarial (in this case, increasing the divergence), then the model is updated to minimize this divergence. In our paper, this process helps enforce the cluster assumption on the target data.
